internal/network/utils_test.go

```diff
@@ -1012,3 +1013,40 @@ var _ = Describe("GetVips", func() {
 		})
 	})
 })
+var _ = Describe("FindInterfaceByIP", func() {
+	var allInterfaces []*models.Interface
+	var searchedInterface models.Interface
+
+	BeforeEach(func() {
+		searchedInterface = models.Interface{
+			IPV4Addresses: []string{"192.168.1.3/24"},
+			IPV6Addresses: []string{"1001:db8:0:200::78/48", "1001:db8:0:200::79/48"},
+		}
+		allInterfaces = []*models.Interface{
+			{
+				IPV4Addresses: []string{"192.168.1.1/24"},
+			},
+			{
+				IPV6Addresses: []string{"1001:db8:0:200::80/48"},
+			},
+			&searchedInterface,
+		}
+
+	})
+	DescribeTable("FindInterfaceByIP", func(allInterfaces *[]*models.Interface, lookForIp string, expectedInterface *models.Interface, expectedError string) {
+		int, err := FindInterfaceByIPString(lookForIp, *allInterfaces)
+		if expectedError == "" {
+			Expect(err).To(BeNil())
+			Expect(int).To(Equal(expectedInterface))
```

Comment:
let's add debug output for the expect command
```suggestion
			Expect(err).To(BeNil(), fmt.Sprintf(err))
			Expect(int).To(Equal(expectedInterface),
			    fmt.Sprintf("expected: %s to be in interface %s", int , expectedInterface ))
```

Reply:
the issue is to avoid the truncation ?

Reply:
so the message will only be shown if the test will fail -- so it just for debug on failure

Reply:
I don't understand the point, the test wil already output the diff with the name of the test. Why adding one more output `expected: %s to be in interface %s"` ?

Reply:
just had a look, the report looks like that in case of failure:
```
$ SKIP_UT_DB=1 go test ./internal/network -ginkgo.focus "FindInterfaceByIP"
Running Suite: Machine network cider Suite
==========================================
Random Seed: 1731427695
Will run 5 of 218 specs

SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS•
------------------------------
• Failure [0.000 seconds]
FindInterfaceByIP
/home/agentil/Projects/openshift/assisted-service/internal/network/utils_test.go:1016
  FindInterfaceByIP
  /home/agentil/Projects/openshift/assisted-service/internal/network/utils_test.go:1036
    Look for IPv6 [It]
    /home/agentil/Projects/openshift/assisted-service/internal/network/utils_test.go:1047

    Expected
        <*errors.errorString | 0xc000583d10>: 
        Cannot find the network interface on which the IP 1001:db8:0:200::75 is set
        {
            s: "Cannot find the network interface on which the IP 1001:db8:0:200::75 is set",
        }
    to be nil

    /home/agentil/Projects/openshift/assisted-service/internal/network/utils_test.go:1039
------------------------------
•••SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Summarizing 1 Failure:

[Fail] FindInterfaceByIP FindInterfaceByIP [It] Look for IPv6 
/home/agentil/Projects/openshift/assisted-service/internal/network/utils_test.go:1039

Ran 5 of 218 Specs in 0.001 seconds
FAIL! -- 4 Passed | 1 Failed | 0 Pending | 213 Skipped
--- FAIL: TestMachineNetworkCidr (0.03s)
FAIL
FAIL	github.com/openshift/assisted-service/internal/network	0.105s
FAIL

```

Reply:
in this case it's not required but, when using `tables` or loops than you don't know what vaules have failed 
but let's take it offline i'll show some examples